src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -396,30 +396,31 @@ async function fetchRegistryBearerToken(params: CommonParams, ociRef: OCIRef | O
 		};
 	}
 
-	let authReq: Buffer;
-	try {
-		authReq = await request(httpOptions, output);
-	} catch (e: any) {
-		// This is ok if the registry is trying to speak Basic Auth with us.
-		output.write(`[httpOci] Could not fetch bearer token for '${service}': ${e}`, LogLevel.Error);
-		return;
+	let res = await requestResolveHeaders(httpOptions, output);
+	if (res.statusCode === 401 || res.statusCode === 403) {
+		output.write(`[httpOci] Credentials for '${service}' may be expired: ${res.resBody.toString()?.trimEnd()}`, LogLevel.Warning);
```

Comment:
Only log res.statusCode and res.statusMessage at LogLevel.Info to avoid too much verbosity since this is a regular case? (The user reported everything was working normally otherwise.)

Reply:
I reduced one LogLevel down, does that look good to you?

Reply:
Thoughts? 
<img width="1057" alt="image" src="https://github.com/devcontainers/cli/assets/23246594/a2dcf72d-0e68-4850-b5e9-a1a22f310275">

Reply:
Sounds good. What about including `res.statusCode` and `res.statusMessage` in the log message for completeness? (The body might be empty, I guess.)